redfish-core/lib/event_service.hpp

```diff
@@ -420,6 +420,54 @@ inline void requestRoutesEventDestinationCollection(App& app)
                 return;
             }
 
+            /* Check snmpTrap port.
+            1. Allow empty port, eg:
+            "snmp://9.41.166.76:".
+            2. Invalid ports and out of range ports are not allowed, eg:
+            "snmp://9.41.166.76:ab"
+            "snmp://9.41.166.76:-40"
+            "snmp://9.41.166.76:65536".
+            */
+
+            if (protocol == "snmp")
```

Comment:
This doesn't do anything. It didn't do anything for 1110. Let's just abandon and mark the spreadsheet as not needed